jupyterhub/templates/hub/configmap.yaml

```diff
@@ -38,6 +38,11 @@ data:
   {{- end }}
   {{- end }}
 
+  {{ if eq .Values.auth.type "custom" -}}
+  auth.custom.class_name: {{ .Values.auth.custom.className | quote }}
```

Comment:
Should be 'class-name'. The naming scheme is confusing since dashes aren't valid in the helm yaml but the convention seems to be to use them in the configmap...